### PR TITLE
pythonPackages.aioprocessing: init at 1.0.1

### PR DESCRIPTION
**Pull request by uskudnik**
###### Motivation for this change
Add aioprocessing module

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by symphorien** [comment]
did you try setting `doCheck = true;` ?

File: pkgs/development/python-modules/aioprocessing/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1yq1gfsky2kjimwdmzqk893sp6387vbl4bw0sbha5hl6cm3jp5dn";
+  };
+
+  meta = {
+    description = "A library that integrates the multiprocessing module with asyncio";
+    homepage = https://github.com/dano/aioprocessing;
+    license = with lib.licenses; [ bsd2 ];
```
Comment (dotlambda) [inline pkgs/development/python-modules/aioprocessing/default.nix]:
```suggestion
    license = lib.licenses.bsd2;
```

**Review by dotlambda — COMMENTED** [review]
Please set `doCheck = false` and add a comment with the reason.

Btw: https://github.com/dano/aioprocessing/pull/25

**Comment by uskudnik** [comment]
@dotlambda Is there a reason for adding `doCheck = false`? Per [manual entry](https://nixos.org/nixpkgs/manual/#ssec-check-phase) it seems to be by default false anyway?

Anyway, included it anyway to make it explicit.

**Comment by dotlambda** [comment]
For `buildPython*`, `doCheck = true` is the default.

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python2.pkgs.aioprocessing python3.pkgs.aioprocessing

**Pull request merged**